demos/linear_mixed_system/linear_mixed_system.py.rst

```diff
@@ -0,0 +1,328 @@
+Linear mixed fluid-structure system
+=================================================
+
+
+.. rst-class:: emphasis
+
+This tutorial is aimed to demonstrate the use of subdomain functionality and show how to describe a system consisting of multiple materials in Firedrake.
+
+The tutorial was contributed by `Tomasz Salwa <mailto:mmtjs@leeds.ac.uk>`__
+and `Onno Bokhove <mailto:O.Bokhove@leeds.ac.uk>`__.
+
+The work is based on the article "Variational modelling of wave-structure interactions with an
+offshore wind-turbine mast" by Tomasz Salwa, Onno Bokhove and Mark Kelmanson :cite:`Salwa:2017`. The authors gratefully acknowledge funding from European Commission, Marie Curie Actions - Initial Training Networks (ITN), project number 607596.
```

Comment:
Please move lines 12 and 13 to the bottom of the page. They are not the first thing that a reader needs to see.